src/Workspaces/Core/Portable/WorkspacesResources.resx

```diff
@@ -613,4 +613,94 @@
   <data name="Changing_document_property_is_not_supported" xml:space="preserve">
     <value>Changing document properties is not supported</value>
   </data>
+  <data name="Alternation_conditions_cannot_be_comments" xml:space="preserve">
```

Comment:
These values are all copied from corefx.  I think the phrasing of many of them is quite poor (and some are downright misleading depending on what the user actually wrote).  However, they match the errors the native parser produces, which i think is quite important.